api/v1alpha1/hosts.go

```diff
@@ -105,3 +110,7 @@ func (hs *HostCollection) Hostnames() []string {
 func (hs *HostCollection) Count() int {
 	return len(hs.hosts)
 }
+
+func IsInternal(hostname string) bool {
```

Comment:
Do we need/want to export this function?

```suggestion
func isInternal(hostname string) bool {
```